src/types.ts

```diff
@@ -142,11 +142,24 @@ export type TransactionRequest = {
     chainId?: unknown,
 }
 
+export interface EventData {
+  address: string,
+  blockHash: string,
+  blockNumber: string,
+  data: string,
+  logIndex: string,
+  removed: boolean,
+  topics: string[],
+  transactionHash: string,
+  transactionIndex: string
+};
+
 /**
  * @see https://github.com/ethers-io/ethers.js/blob/v5.0.4/packages/abstract-provider/src.ts/index.ts#L224
  */
 export interface EthersProvider {
   call(transaction: TransactionRequest, blockTag?: never): Promise<string>;
+  getLogs(filter): Promise<any>;
```

Comment:
The original one looks a bit different

```
getLogs(filter: Filter): Promise<Array<Log>>;
```

I wonder now, can we use type-only imports from a dev dependency?